### ISSUE TITLE
Scaling GMM to massive datasets

### ISSUE THREAD

**Issue by yangzhaopeng3** [ldeecke/gmm-torch#18]
Hi Lucas,
Thanks for your repo, it helped me a lot. But when I am using gmm-torch in my research, a tricky issue occured. I need to fit a GMM in a very large dataset(e.g. ImageNet), which can not be loaded in memory at once. So I am wondering if there is a way to handle this issue? Can GMM be fitted in batch training setting or if we can use gradient descend to estimate GMM's parameters?
Thanks.
